cpu/cc26x0/periph/rfc.c

```diff
@@ -7,37 +7,41 @@
  */
 #include <stdio.h>
 #include <string.h>
+#include <inttypes.h>
```

Comment:
this comment is about the whole commit ;-)
there's still a lot going on in here.

how about:
```
commit 81e8ee99656b10a79097ec63fa14e89461863053
Author: Florent-Valéry <florent.coen@gmail.com>
Date:   Thu Apr 21 14:08:53 2016 +0200

    cpu/cc26x0: enable interrupts of RF Core in NVIC

diff --git a/cpu/cc26x0/periph/rfc.c b/cpu/cc26x0/periph/rfc.c
index c88134c..152f39c 100644
--- a/cpu/cc26x0/periph/rfc.c
+++ b/cpu/cc26x0/periph/rfc.c
@@ -14,6 +14,14 @@
 
 #define BLE_ADV_STR "this is not a riot\n"
 
+void rfc_irq_enable(void)
+{
+    NVIC_EnableIRQ(RF_CMD_ACK_IRQN);
+    NVIC_EnableIRQ(RF_CPE0_IRQN);
+    NVIC_EnableIRQ(RF_CPE1_IRQN);
+    NVIC_EnableIRQ(RF_HW_IRQN);
+}
+
 void isr_rfc_cmd_ack(void)
 {
     printf("cmd_ack %lx\n", RFC_DBELL->RFACKIFG);
@@ -106,6 +114,8 @@ void rfc_prepare(void)
 
     printf("rfc-prep\n");
 
+    rfc_irq_enable();
+
     rfc_setup_ble();
 
         rfc_beacon();
```

i would probably allow `rfc_irq_disable` sneak into the commit^^
but it isn't used anywhere.